### PR TITLE
first draft add missing endpoint idps

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Adding the endpoint to get a user by idp/user ID

### Resolves:
[OKTA-85733](https://oktainc.atlassian.net/browse/OKTA-85733)

### CODE REVIEW

File: _source/_docs/api/resources/idps.md [hunk 1]
```diff
@@ -3560,7 +3560,7 @@ curl -v -X GET \
 
 ## Identity Provider User Operations
 
-### Listing IdPs associated with a user
+### List IdPs Associated with a User
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/idps.md]: Just a style guide change

File: _source/_docs/api/resources/idps.md [hunk 2]
```diff
@@ -3694,14 +3694,163 @@ Content-Type: application/json
...
+  "errorSummary": Not found: Resource not found: 0oa62bfdiumsUndnZ0h8 (IdpAppInstance)",
   "errorLink": "E0000007",
   "errorId": "oaeYW9k9yJuSSSkhaMQdA1-Zg",
   "errorCauses": []
 }
 ~~~
 
-### Link a user to a Social IdP without a transaction
+### Link a User to a Social IdP Without a Transaction
 {:.api .api-operation}
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/idps.md]: Just a style guide change.

File: _source/_docs/api/resources/idps.md [hunk 3]
```diff
@@ -3694,14 +3694,163 @@ Content-Type: application/json
 
 {
   "errorCode": "E0000007",
-  "errorSummary": "Not found: Resource not found: 00u3b8ixnOCwbn8Sn0g4 (User)",
+  "errorSummary": "Not found: Resource not found: 00ub0oNGTSWTBKOLGLNR (User)",
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/idps.md]: Git interpreted my changes oddly. There's no real change until line 3704.

**Comment by mystiberry-okta** [comment]
Either @yuliu-okta or @danielzeller-okta  for review?

File: _source/_docs/api/resources/idps.md [hunk 4]
```diff
@@ -3694,14 +3694,163 @@ Content-Type: application/json
...
+
+##### Request Example
+{:.api .api-request .api-request-example}
+
+~~~sh
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+    'https://{yourOktaDomain}.com/api/v1/idps/0oa62bfdiumsUndnZ0h7/users?expand=user' \
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/idps.md]: missing uid and it does not support expand

Comment (mystiberry-okta) [reply _source/_docs/api/resources/idps.md]: I copied the wrong postman request, sorry! fixing now.

File: _source/_docs/api/resources/idps.md [hunk 5]
```diff
@@ -3730,13 +3729,14 @@ curl -v -X GET \
...
 -H "Authorization: SSWS ${api_token}" \
-    'https://{yourOktaDomain}.com/api/v1/idps/0oa62bfdiumsUndnZ0h7/users?expand=user' \
+    'https://{yourOktaDomain}.com/api/v1/idps/0oa62bfdiumsUndnZ0h7/users/00u5t60iloOHN9pBi0h7' \
 ~~~
 
 ##### Response Example
 {:.api .api-response .api-response-example}
 
 ~~~json
+[
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/idps.md]: I left out the brackets before, but put them in this time. Confusing, though, as I thought [] indicated an array, but this endpoint should only ever give one result, shouldn't it? Anyway, just wanted to mention that.

Comment (mystiberry-okta) [reply _source/_docs/api/resources/idps.md]: Ignore this draft--I pasted in the wrong result.

**Pull request merged**